Makefile

```diff
@@ -28,14 +38,15 @@ help:
 	@echo " * 'lint' - Execute the source code linter"
 	@echo " * 'gofmt' - Verify the source code gofmt"
 
-.PHONY: check-gopath
+.gopathok:
+	if [ ! -d "$(GOPKGDIR)" ]; then \
```

Comment:
I think you can use `ifdef` here? http://stackoverflow.com/questions/8811526/using-conditional-rules-in-a-makefile